connect_transformations/vat_rate/mixins.py

```diff
@@ -24,18 +24,20 @@ def preload_eu_vat_rates(self):
             self.eu_vat_rates = {}
 
             try:
-                url = 'https://api.exchangerate.host/vat_rates'
-                response = requests.get(url)
+                url = 'https://api.apilayer.com/tax_data/rate_list'
+                response = requests.get(
+                    url,
+                    headers={'apikey': self.config['EXCHANGE_API_KEY']},
+                )
                 response.raise_for_status()
+
                 data = response.json()
-                if not data['success']:
-                    raise VATRateError(
-                        f'Unexpected response calling {url}',
-                    )
-                for key, rate in data['rates'].items():
-                    value = rate['standard_rate']
-                    self.eu_vat_rates[key] = value
-                    self.eu_vat_rates[rate['country_name']] = value
+                for rate in data['data']:
+                    if not rate['eu']:
+                        continue
+                    value = rate['standard_rate']['rate']
+                    self.eu_vat_rates[rate['country_code']] = int(value * 100)
```

Comment:
why int ?

Reply:
in previous version vat rates were a integer numbers, so for backward compatibility made from values 0.XX -> XX